radio/src/translations/en.h.txt

```diff
@@ -925,6 +925,8 @@
 #define TR_INPUTS              "Inputs"
 #define TR_OUTPUTS             "Outputs"
 #define TR_EEBACKUP            "\004[ENTER Long] to backup the EEPROM"
+#define TR_FACTORYRESET        "\012[MENU Long]: Factory reset"
```

Comment:
Yes the confirmation is there, but not a contextual menu. I am not sure what is best. Perhaps it's good to keep this as it is, but if we have a new entry to add, we put everything in a menu.